dbt_project/tests/test_stg_ocsf_findings_status_names.sql

```diff
@@ -0,0 +1,8 @@
+-- Test that finding_status only contains valid OCSF status names
```

Comment:
Same as finding_severity, these are standard OCSF statuses, so we can reuse the same test macro for both upstream and downstream. We can move the `dbt_project/macros/test_status.sql` macro here and reference it from the schemas in both locations. Something like:
```
- name: finding_status
      data_tests:
        - status_name
```